pkg/environment/v1/environment.go

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package environment
+package v1
```

Comment:
I think this was always just `environment`, because no v2 environment package exists - they're just part of the manifest in v2. 
Do you plan to move v2 environment related things to the package in the future? Or just added v1 to align with other packages?

Reply:
>  they're just part of the manifest in 
Actually, no it is not used in Manifest v2. that's why i moved it to v1. like it was done for e.g. project v1.

Reply:
> no it is not used in Manifest v2

I meant to say that the concept of `environment` in v2 is part of the manifest directly. 

So unlike `config` and `project` there is no v2 code for the `environment` package, which is why there was no sub-package so far. 
If there's no intention to change that, the package could just become `v1environment` instead of adding a sub-package with a non-explicit import name

Reply:
I've moved environment stuff into "v1environment" package inside convert